user/views.py

```diff
@@ -4,15 +4,17 @@
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
 
-from user.serializers import UserSerializer
+from user.serializers import UserSerializer, AuthTokenSerializer
 
 
 class CreateUserView(generics.CreateAPIView):
     serializer_class = UserSerializer
+    permission_classes = ()
```

Comment:
The `permission_classes` for `CreateUserView` is set to an empty tuple, which means no permissions are required to access this view. Ensure that this is intentional and aligns with your application's security requirements.